client/app/telecom/dashboard/translations/Messages_fi_FI.xml

```diff
@@ -20,6 +20,7 @@
    <translation id="telecom_dashboard_FAQ_Activate" qtlid="305018">Palvelujen aktivointi</translation>
    <translation id="telecom_dashboard_FAQ_managed_modem" qtlid="305031">Modeemin konfigurointi</translation>
    <translation id="telecom_dashboard_FAQ_init_modem" qtlid="305044">Modeemin nollaus</translation>
+   <translation id="telecom_dashboard_FAQ_restore_service">Palauta palvelusi</translation>
```

Comment:
You don't have to send the translations, our translator team will handle everything. Plus, those files will be erased by our translation system when building, so there's no need to provide other texts than French.